packages/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -44,7 +44,7 @@ export const usePresenceBadge = (props: PresenceBadgeProps, ref: React.Ref<HTMLE
       {
         size: 'medium',
         ...props,
-        icon: resolveShorthand(undefined, {
+        icon: resolveShorthand((undefined as unknown) as IntrinsicShorthandProps<'span'>, {
```

Comment:
Does this work instead?
```suggestion
        icon: resolveShorthand(undefined as (IntrinsicShorthandProps<'span'> | undefined), {
```